pkg/tsdb/cloudwatch/log_actions.go

```diff
@@ -164,15 +165,15 @@ func (e *cloudWatchExecutor) handleGetLogEvents(ctx context.Context, logsClient
 	}
 
 	messages := make([]*string, 0)
-	timestamps := make([]*int64, 0)
+	timestamps := make([]time.Time, 0)
```

Comment:
hmmm everything is normally a pointer. I think we'd better make this a slice of pointers as well. going to make that change in a commit with a test as well, feel free to change if you'd like

Reply:
Actually I'm not really sure, let me know what you think.

Reply:
I mostly see pointers because of the type of response from the API usually, maybe it is not necessary here for the frame fields

Reply:
Hm, Grafana wide non-pointers are preferred for time slices I think. I think both is fine 🤷

Reply:
great!